apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -744,7 +744,7 @@ const BookingPage = ({
                           </div>
                         </div>
                       )}
-                      {input.options && input.type === EventTypeCustomInputType.RADIO && (
+                      {input.options && input.type === EventTypeCustomInputType.BOOL && (
```

Comment:
Is this right? Below is showing the radio inputs - not sure why its been changed to bool

Reply:
Also the input.options below only is available on RADIO type